test-utils/port.go

```diff
@@ -0,0 +1,19 @@
+package e2eutils
+
+import (
+	"fmt"
+	"net"
+)
+
+func IsPortAvailable(port int) bool {
+	return IsHostPortAvailable("127.0.0.1", port)
+}
+
+func IsHostPortAvailable(host string, port int) bool {
+	listener, err := net.Listen("tcp", fmt.Sprintf("%s:%d", host, port))
+	if err != nil {
+		return false
+	}
+	listener.Close()
```

Comment:
Need to add `defer` and move to line 14, otherwise we not free port in case if it not occupied